src/stars__stars.css

```diff
@@ -0,0 +1,17 @@
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Good work, just remove margin-right from the last stars in a row, use the selector below:

.stars__star:last-child {
  margin-right: 0;
}

Reply:
after removing this line tests fail